spec/uap_clj/common_spec.clj

```diff
@@ -1,3 +1,27 @@
-(ns uap-clj.common-spec)
+(ns uap-clj.common-spec
+  (:require [clojure.java.io :as io :refer [resource]]
+            [clj-yaml.core :refer [parse-string default-loader-options]])
```

Comment:
I wonder if you need that refer, since you only use the fully qualified symbol for `default-loader-options`